src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Workspace/Mef/IMefHostExportProvider.cs

```diff
@@ -12,4 +13,13 @@ internal interface IMefHostExportProvider
         IEnumerable<Lazy<TExtension, TMetadata>> GetExports<TExtension, TMetadata>();
         IEnumerable<Lazy<TExtension>> GetExports<TExtension>();
     }
+
+    internal static class IMefHostExportProviderExtensions
```

Comment:
❔ Are these needed in production code, or is it good enough to keep them in test code?

Reply:
Moved to test utils.